setup.py

```diff
@@ -1,9 +1,12 @@
 from setuptools import setup, find_packages
 
 
-version = '0.1.2'
+version = '0.1.3'
 
-requires = []
+requires = [
+    'pysaml2>=1.0.0',
+    'redis>=2.10.5',
```

Comment:
hmm, I don't think we will want to draw in pysaml2 and redis everywhere eduid-common is used? not sure if we've 'scoped' eduid-common to being for web apps...